### PR TITLE
NTGR-608: Sanitize all request inputs

### PR DESCRIPTION
**Pull request by kei178**
This PR adds sanitization to all request inputs removing all `phpcs:ignore WordPress.Security.ValidatedSanitizedInput.InputNotSanitized` from the code.

### CODE REVIEW

File: includes/class-accredible-learndash-admin-menu.php [hunk 1]
```diff
@@ -136,7 +114,7 @@ public static function add_action_links( $links ) {
 			$mylinks = array(
 				'<a href="' . admin_url( 'admin.php?page=accredible_learndash_settings' ) . '">Settings</a>',
 			);
-			return array_merge( $links, $mylinks );
+			return array_merge( $mylinks, $links );
```
Comment (kei178) [inline includes/class-accredible-learndash-admin-menu.php:117]: To change the order of the links on the plugin page to "Settings, Deactivate".

Screen Shot 2022-03-30 at 15 20 58

File: includes/models/class-accredible-learndash-model.php [hunk 2]
```diff
@@ -34,7 +34,7 @@ public static function get_results( $where_sql = '', $limit = null, $offset = nu
 				$sql .= " OFFSET $offset";
 			}
 			// XXX `$where_sql` is a raw SQL so `$wpdb->prepare` cannot be used.
-			return $wpdb->get_results( $sql ); // phpcs:disable WordPress.DB.PreparedSQL.NotPrepared
+			return $wpdb->get_results( $sql ); // phpcs:ignore WordPress.DB.PreparedSQL.NotPrepared
```
Comment (kei178) [inline includes/models/class-accredible-learndash-model.php:37]: This change is not directly related to this task, but we need to use `phpcs:ignore` for inline comments unless adding `phpcs:enable` after `phpcs:disable`.

**Review by octaviorequena — APPROVED** [review]
Nice work @kei178 ! I have tested all the functionality and it's working as expected.

**Review by Ndibui — APPROVED** [review]
